### DISCUSSION TITLE
GHES Renovate - Scan repositories by topics in organization

### DISCUSSION THREAD

**Discussion by saggenerics** [renovatebot/renovate#13423] [Request Help]
Hello,

Is it possible to configure Renovate Self-Hosted in such a way, that repositories don't need to be listed in the config, but rather everyone who wants to have Renovate scan the dependencies and update them, have to add a topic to the repository? That way it is very easy for everyone in an organization to use Renovate without needing to edit the base Renovate configuration.

Best Regards,
Kristiyan

**Comment by rarkins** [comment]
No, it's not possible. But why doesn't it work that everyone who wants Renovate in a repo just adds the renovate user to the repo?
